### PR TITLE
Tx manager

### PR DESCRIPTION
**Pull request by moskalyk**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
   ],
   "scripts": {
     "tsc": "tsc -p tsconfig.json",
-    "dev": "tsx src/bin.ts",
+    "remove-built-files": "node removeJsFiles.js",
```
Comment (taylanpince) [inline package.json]: @moskalyk Can you remove this one, and the `removeJsFiles` – I am not comfortable with it and I don't think it's needed.

**Review by taylanpince — CHANGES_REQUESTED** [review]
Only minor nit

File: package.json (hunk 1 above)
Comment (moskalyk) [reply package.json]: Yeah sure

**Review by taylanpince — CHANGES_REQUESTED** [review]
@moskalyk One more thing that's bugging me: Why do we need dev mode? CLI would by definition only run in dev mode so I think we can remove that distinction.

**Review by taylanpince — APPROVED** [review]

**Pull request merged**